meta-lmp-base/recipes-connectivity/bluez5/bluez5_5.65.bbappend

```diff
@@ -0,0 +1,16 @@
+# This enables using BSD library libedit for client/mesh features
+
+FILESEXTRAPATHS:prepend := "${THISDIR}/files:"
+
+READLINE_LIB = "${@bb.utils.contains('LMP_DISABLE_GPLV3', '1', 'libedit', 'readline', d)}"
```

Comment:
I think it is better to have this remove/append logic in meta-lmp-base/classes/lmp-disable-gplv3.bbclass, as a way to isolate the selection of the options used for configure.